_posts/2017-12-08-alpha1.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Alpha, la 1ère formation à destination des acteurs de l’innovation du secteur public 
+authors: jihane.herizi
+---
+
+
+# Alpha : la 1ère formation à destination des acteurs de l’innovation du secteur public 
+
+Après avoir [trouvé nos intrapreneurs de startup d’État](https://beta.gouv.fr/2017/02/27/comment-former-des-intrapreneurs.htmlÁ) nous vous expliquions, dans un précédent article, [comment ils sont quotidiennement formés en interne](https://beta.gouv.fr/2017/02/27/comment-former-des-intrapreneurs.html), notamment grâce à leurs coachs respectifs et aux autres membres des Startups d’Etat. 
+
+Pour aller plus loin, et parce que les Startups d’État et donc leurs intrapreneurs, vont être de plus en plus nombreux dans les mois et années à venir, il était nécessaire qu’une formation accélérée sur les méthodes de l’incubateur Beta.gouv. voit le jour. 
+
+C’est ainsi qu’Alpha est née le 7 novembre 2017 et a accueilli pendant 3 jours, dans les locaux de la Fabrique Pôle Emploi, 28 acteurs de l’innovation du secteur public, membres d’une Startup d’État ou désireux d’en créer une. 
+
+Avec eux, 14 intervenants coachs, intrapreneurs, développeurs, business développeurs venus les outiller sur des sujets tels que  « Comment trouver son premier client » à « Écouter ses utilisateurs partout le temps » en passant par « Survivre en milieu hostile » ou « Comment faire de l’industriel en mode artisanal ».
+
+Cliquez ici pour (re)vivre en quelques tweets les temps forts de la formation ! 
+
+Interventions en plénière, alternée d’ateliers concrets facilités par du coaching en direct, ont permis à cette première promotion issue des quatre coins de la France et de l’administration (La CNAMTS, les Ministères des Affaires Sociales ou de l’Écologie, la DILA, les départements du Calvados, de l’Isère et du Pas de Calais, etc.) de ressortir en confiance et (re)boostés. 
+
+Cliquez ici pour découvrir l’album photo de cet première édition !
+
+Simplification des autorisations de travail en France, réseau des A+.... dès le lundi suivant, certains d’entre eux mettaient déjà en place les premiers enseignements pour faire éclore ou accélérer leur produit. 
```

Comment:
Ici je dois donner des exemples de startups présentes à la formation (work in france, A+..) il y avait MRS aussi non ? d'autres startups svp ?